File: java/basic_app/app/src/main/java/com/kuku/muku/kukumukubasicapp/BananasActivity.java
package com.kuku.muku.kukumukubasicapp;

import android.os.Bundle;

public class BananasActivity extends FruitActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setStaticAttributes("bananas");
    }

    @Override
    protected int getLayoutResourceId() {
        return R.layout.activity_bananas;
    }
}

File: kotlin/basic_app/app/src/main/java/com/example/onelinkbasicapp/BananasActivity.kt
package com.example.onelinkbasicapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class BananasActivity : FruitActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }
    override protected fun getLayoutResourceId():Int{
        return R.layout.activity_bananas;
    }
}